superset-frontend/src/components/ObjectTags/ObjectTags.css

```
@@ -0,0 +1,149 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+.ant-tag {
+  color: black;
+}
+
+.react-tags {
+  position: relative;
+  display: inline-block;
+  padding: 1px 0 0 1px;
+  margin: 0 10px;
+  border: 0px solid #f5f5f5;
+  border-radius: 1px;
+
+  /* shared font styles */
+  font-size: 12px;
+  line-height: 1.2;
+
+  /* clicking anywhere will focus the input */
+  cursor: text;
+}
+
+.react-tags__selected {
+  display: inline;
+}
+
+.react-tags__selected-tag {
+  display: inline-block;
+  box-sizing: border-box;
+  margin: 0;
+  padding: 6px 8px;
+  border: 0px solid #f5f5f5;
+  border-radius: 2px;
+  background: #f1f1f1;
+
+  /* match the font styles */
+  font-size: inherit;
+  line-height: inherit;
+}
+
+.react-tags__search {
+  display: inline-block;
+
+  /* new tag border layout */
+  border: 1px dashed #d9d9d9;
+
+  /* match tag layout */
+  line-height: 20px;
+  margin-bottom: 0;
+  padding: 0 7px;
+
+  /* prevent autoresize overflowing the container */
+  max-width: 100%;
+}
+
+.react-tags__search:focus-within {
+  border: 1px solid #000000;
+}
+
+@media screen and (min-width: 30em) {
+  .react-tags__search {
+    /* this will become the offsetParent for suggestions */
+    position: relative;
+  }
+}
+
+.react-tags__search input {
+  max-width: 150%;
+
+  /* remove styles and layout from this element */
+  margin: 0;
+  margin-left: 0;
+  padding: 0;
+  border: 0;
+  outline: none;
+
+  /* match the font styles */
+  font-size: inherit;
+  line-height: inherit;
+}
+
+.react-tags__search input::-ms-clear {
+  display: none;
+}
+
+.react-tags__suggestions {
+  position: absolute;
+  top: 100%;
+  left: 0;
+  width: 100%;
+  z-index: 9999;
+}
+
+@media screen and (min-width: 30em) {
+  .react-tags__suggestions {
+    width: 240px;
+  }
+}
+
+.react-tags__suggestions ul {
+  margin: 4px -1px;
+  padding: 0;
+  list-style: none;
+  background: white;
+  border: 1px solid #d1d1d1;
+  border-radius: 2px;
```

```suggestion
  border-radius: ${({ theme }) => theme.borderRadius}px;
```
Border-radius values should stay within the Superset theme, the current value is 4 as seen [here](https://github.com/apache/superset/blob/master/superset-frontend/packages/superset-ui-core/src/style/index.tsx#L55).